extra_data/components/multimod_det.py

```diff
@@ -8,13 +8,14 @@
 import numpy as np
 import pandas as pd
 
-from .exceptions import SourceNameError
-from .reader import DataCollection, by_id, by_index
-from .read_machinery import DataChunk, roi_shape, split_trains
-from .writer import FileWriter
-from .write_cxi import XtdfCXIWriter, JUNGFRAUCXIWriter
+from ..exceptions import SourceNameError
+from ..reader import DataCollection, by_id, by_index
+from ..read_machinery import DataChunk, roi_shape, split_trains
+from ..writer import FileWriter
+from ..write_cxi import XtdfCXIWriter, JUNGFRAUCXIWriter
 
 __all__ = [
+    'XtdfDetectorBase',
```

Comment:
I added this class here, as it was the only symbol used outside of the `components` and in particular also referenced on docstrings there.